src/jaguar.toit

```diff
@@ -26,7 +26,10 @@ RUN_DEFINES_HEADER ::= "X-Jaguar-Run-Defines"
 
 logger ::= log.Logger log.INFO_LEVEL log.DefaultTarget --name="jaguar"
 validate_firmware / bool := firmware.is_validation_pending
-disabled / monitor.Signal? := null
+
+disabled / bool := false
```

Comment:
Add a comment what this global does.
It looks like it's used as an indication that Jaguar should disable itself, and not that it is already disabled.
Maybe a better name would be `program_needs_full_network` or something like this.

Reply:
Done.